test/test_spspmm.py

```diff
@@ -35,7 +35,7 @@ def test_sparse_tensor_spspmm(dtype, device):
         ], dtype=dtype, device=device),
     )
 
-    expected = torch.eye(10, dtype=dtype, device=device)
+    expected = torch.eye(10, device=device).to(dtype)
```

Comment:
What's the reason for this change?

Reply:
`eye` doesn't have BFloat16 implementation.